packages/app-desktop/gui/ConfigScreen/controls/plugins/FilterForPlugins/FilterForPlugins.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { themeStyle } from '@joplin/lib/theme';
+import uuid from '@joplin/lib/uuid';
+
+interface FilterProps {
+    themeId: number;
+    setFilterValue: Function;
+	onSearchButtonClick: Function;
+	filterValue: string;
+}
+
+export default function FilterForPlugins(props: FilterProps) {
+
+	const theme = themeStyle(props.themeId);
+
+	const filterOptions = ['Select plugin category','Most Downloaded','Recommended', 'Newest', 'Built-in', 'All', 'line-separator', 'Appearance', 'Developer tools', 'Productivity', 'Themes', 'Integrations', 'Viewer', 'Search', 'Tags', 'Editor', 'Files', 'Personal knowledge management', 'line-separator', 'Installed', 'Enabled', 'Disabled', 'Outdated'];
```

Comment:
What is this thing? Why are the options defined as user-facing strings? How about localisation? And the list of categories being copied and pasted again......

So to be clear, you need to split the Select options into value (for programmatic use - eg. "most_downloaded", "newest", etc.) and text (user-facing string).